x-pack/plugins/ml/public/components/anomalies_table/anomalies_table_columns.js

```diff
@@ -0,0 +1,257 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+
+
+import {
+  EuiButtonIcon,
+  EuiHealth,
+  EuiLink,
+} from '@elastic/eui';
+
+import React from 'react';
+import _ from 'lodash';
+
+import {
+  formatHumanReadableDate,
+  formatHumanReadableDateTime,
+  formatHumanReadableDateTimeSeconds
+} from '../../util/date_utils';
+
+import { DescriptionCell } from './description_cell';
+import { DetectorCell } from './detector_cell';
+import { EntityCell } from './entity_cell';
+import { InfluencersCell } from './influencers_cell';
+import { LinksMenu } from './links_menu';
+import { checkPermission } from '../../privilege/check_privilege';
+import { mlFieldFormatService } from '../../services/field_format_service';
+import { getSeverityColor, isRuleSupported } from '../../../common/util/anomaly_utils';
+import { formatValue } from '../../formatters/format_value';
+
+
+// Maximum number of influencers to display before a 'show more' link is added.
+export const INFLUENCERS_LIMIT = 5;
+
+export const ANOMALIES_TABLE_TABS = {
+  DETAILS: 0,
+  CATEGORY_EXAMPLES: 1
+};
+
+function renderTime(date, aggregationInterval) {
+  if (aggregationInterval === 'hour') {
```

Comment:
It might be an idea to export the `OPTIONS` const from `public/components/controls/select_interval/select_interval.js` and use those values rather than hardcoded `hour` and `second`.

Reply:
🤔 I'd usually agree but, in this case the imported `OPTIONS` would have to be used like
`if (aggregationInterval === OPTIONS[1].value)`
which I think is harder to read and is dependent on the order of the options.